lib/shared/addon/components/form-name-description/component.js

```diff
@@ -18,6 +18,7 @@ export default Component.extend({
   nameHelpText           : '',
   nameRequired           : false,
   nameDisabled           : false,
+  horizontal             : false,
```

Comment:
Can you just tweak the add account form to use the normal (horizontal=false) view so that it's more consistent with other pages?

That way you don't need to add this `horizontal` option (which doesn't support all the description-related options).